features/step_definitions/templates.js

```diff
@@ -3,8 +3,78 @@
 const fs = require('mz/fs');
 const path = require('path');
 const Assert = require('chai').assert;
-const { Given, Then, When } = require('cucumber');
 const request = require('screwdriver-request');
+const { Before, Given, Then, When } = require('cucumber');
+
+const TIMEOUT = 240 * 1000;
+
+Before(
+    {
+        tags: '@templates',
+        timeout: TIMEOUT
+    },
+    function hook() {
+        this.repoOrg = this.testOrg;
+        this.repoName = 'functional-template';
+        this.templateNamespace = 'screwdriver-cd-test';
+        this.branchName = 'master';
+        this.pipelineId = null;
+        this.jwt = null;
+        this.template = null;
+        this.jobName = null;
+
+        this.startJob = jobName => {
+            return this.ensurePipelineExists({
+                repoName: this.repoName,
+                branch: this.branchName,
+                shouldNotDeletePipeline: true
+            })
+                .then(() =>
+                    request({
+                        uri: `${this.instance}/${this.namespace}/events`,
+                        method: 'POST',
+                        json: true,
+                        body: {
+                            pipelineId: this.pipelineId,
+                            startFrom: jobName
+                        },
+                        auth: {
+                            bearer: this.jwt
+                        }
+                    })
+                )
+                .then(response => {
+                    Assert.equal(response.statusCode, 201);
+                    this.eventId = response.body.id;
+                })
+                .then(() =>
+                    request({
+                        uri: `${this.instance}/${this.namespace}/events/${this.eventId}/builds`,
+                        method: 'GET',
+                        auth: {
+                            bearer: this.jwt
+                        },
+                        json: true
+                    })
+                )
```

Comment:
This is the way it was written before. This will cause the request to fail.
Please refer to the following link.
https://github.com/screwdriver-cd/screwdriver/blob/master/features/support/sdapi.js#L198-L203

Reply:
OK, I fixed.